pkg/build/netbsd.go

```diff
@@ -66,3 +73,47 @@ func (ctx netbsd) clean(kernelDir string) error {
 	// machine.
 	return nil
 }
+
+// Copy the compiled kernel to the qemu disk image using ssh
+func CopyKernelToDisk(outputDir string) error {
+	temp := []byte(`
+{
+	"qemu": "qemu-system-x86_64",
```

Comment:
"qemu" value is the default used by vm/qemu/qemu.go, so let's not specify it at all not duplicate things.
"count" is also 1 by default.
"cpu" we probably need to make 1 default and then remove from here.